server/models/User.js

```diff
@@ -446,7 +446,7 @@ export default (Sequelize, DataTypes) => {
 
     const sequelizeParams = transaction ? { transaction } : undefined;
     debug('createUserWithCollective', userData);
-    const cleanUserData = pick(userData, ['email', 'firstName', 'lastName', 'newsletterOptIn']);
+    const cleanUserData = pick(userData, ['email', 'firstName', 'lastName', 'newsletterOptIn', 'data']);
```

Comment:
I'd prefer if we keep the `data` out of users reach, we use this field to store info that they should not be able to alter

Reply:
@Betree what is the actionable?

Reply:
To pass it in a way users can't submit it themselves.

With the current code if you call `/api/singin` with the following payload, it will be accepted:
```js
{ 
  ...userData, 
  lastSignInRequest: { ip: 'fake-ip' }, 
  feature: { allTheFeaturesThatIWant: true } 
}
```

A potential way to solve this can be to change the signature of `createUserWithCollective` to:
```
User.createUserWithCollective = async (userData, {transaction, creationRequest})
```

Or your could also update the user after the creation in the relevant paths.